composer/loggers/logger_hparams.py

```diff
@@ -186,10 +188,12 @@ def get_flattened_dict(data: Dict[str, Any], _prefix: List[str] = []) -> Dict[st
                 self.extra_init_params["config"] = {}
             self.extra_init_params["config"].update(flattened_config)  # type: ignore
 
+        name_suffix = f"Rank{dist.get_global_rank()}"
```

Comment:
```suggestion
        name_suffix = f"Rank {dist.get_global_rank()}"
```

Reply:
I actually wanted no space to match what custom run names usually look like, but I'll add it back in